frontend/integration-tests-cypress/cypress-e2e/cypress/support/commands.js

```diff
@@ -0,0 +1,72 @@
+// ***********************************************
+// This example commands.js shows you how to
+// create various custom commands and overwrite
+// existing commands.
+//
+// For more comprehensive examples of custom
+// commands please read more here:
+// https://on.cypress.io/custom-commands
+// ***********************************************
+//
+//
+// -- This is a parent command --
+// Cypress.Commands.add("login", (email, password) => { ... })
+//
+//
+// -- This is a child command --
+// Cypress.Commands.add("drag", { prevSubject: 'element'}, (subject, options) => { ... })
+//
+//
+// -- This is a dual command --
+// Cypress.Commands.add("dismiss", { prevSubject: 'optional'}, (subject, options) => { ... })
+//
+//
```

Comment:
Should we convert to typescript?

Reply:
Yes, pls read the Open Issues doc in the description -thanks

Reply:
Fixed, converted `commands.js` to `commands.TS`.  
see [comment](https://github.com/openshift/console/pull/4831#issuecomment-611705992)